types/src/transaction/action.rs

```diff
@@ -18,15 +18,14 @@ use crate::errors::SyntaxError;
 use crate::transaction::ShardTransaction;
 use crate::{CommonParams, ShardId, Tracker};
 use ccrypto::Blake;
-use ckey::{recover, Address, NetworkId, Public, Signature};
+use ckey::{recover, Address, NetworkId, Signature};
 use primitives::{Bytes, H256};
 use rlp::{Decodable, DecoderError, Encodable, Rlp, RlpStream};
 
 #[derive(Clone, Copy)]
 #[repr(u8)]
 enum ActionTag {
     Pay = 0x02,
-    SetRegularKey = 0x03,
```

Comment:
@somniumism How about pulling the ActionTag numbers one by one to fill from `0x03`?

Reply:
You mean as follows?
```rust
enum ActionTag {
    Pay = 0x02,
    CreateShard = 0x03,
    SetShardOwners = 0x04,
    SetShardUsers = 0x05,
    ShardStore = 0x19,
    Custom = 0xFF,
}
```

Reply:
or
```rust
enum ActionTag {
    Pay = 0x02,
    CreateShard = 0x03,
    SetShardOwners = 0x04,
    SetShardUsers = 0x05,
    ShardStore = 0x06,
    Custom = 0x07,
}
```

Reply:
I prefer the above one. I guess `Custom` ActionTag was set to `0xFF` intentionally but I'm not sure about `ShardStore`

Reply:
Thank you. I modified it. : )